sdk/src/main/java/io/matchmore/sdk/api/SubscriptionApi.kt

```diff
@@ -0,0 +1,62 @@
+package io.matchmore.sdk.api
```

Comment:
is there any way to automatically mark generated files? maybe simple header comment?

Reply:
Currently, we can't generate kotlin code so all API classes were generated to temp folder and manually(using Android Studio functionality) converted to kotlin. 
Is there any reason to mark that classes?

Reply:
Sounds good. I just want to understand all parts.